CONTRIBUTING.md

```diff
@@ -54,17 +54,8 @@ taken straight from the Ruby on Rails guide:
 * Update the documentation, the surrounding one, examples elsewhere, guides,
   whatever is affected by your contribution
 
-Syntax:
-
-* Two spaces, no tabs.
-* No trailing whitespace. Blank lines should not have any space.
-* Prefer &&/|| over and/or.
-* `MyClass.my_method(my_arg)` not `my_method( my_arg )` or `my_method my_arg`.
-* `a = b` and not `a=b`.
-* `a_method { |block| ... }` and not `a_method { | block | ... }`
-* Follow the conventions you see used in the source already.
-* -> symbol over lambda
```

Comment:
I stick to doing `->(foo) { ... }` as I like to think if it as a 'stabby' arrow :p Not seeing anything in the GH style guide; again potentially superfluous on my part for saying this.